ingestion/src/main/java/feast/store/serving/redis/RedisCustomIO.java

```diff
@@ -189,23 +186,14 @@ public Write withTimeout(int timeout) {
 
     public static class WriteDoFn extends DoFn<RedisMutation, FailedElement> {
 
-      private final String host;
-      private final int port;
-      private final BackOffExecutor backOffExecutor;
       private final List<RedisMutation> mutations = new ArrayList<>();
-
-      private Jedis jedis;
-      private Pipeline pipeline;
       private int batchSize = DEFAULT_BATCH_SIZE;
       private int timeout = DEFAULT_TIMEOUT;
+      private RedisIngestionClient ingestionClient;
```

Comment:
ingestionClient makes this seem generic, but it's actually a Redis specific client. Can we just call it redisClient?

Reply:
👍